corehq/apps/hqwebapp/static/hqwebapp/js/initial_page_data.js

```diff
@@ -39,10 +39,13 @@ hqDefine('hqwebapp/js/initial_page_data', ['jquery', 'underscore'], function ($,
     /*
      * Fetch a named value.
      */
-    var get = function (name) {
+    var get = function (name, strict) {
         if (_initData[name] === undefined) {
             _initData = gather(data_selector, _initData);
         }
+        if (strict && _initData[name] === undefined) {
```

Comment:
Thoughts on using ```in``` instead of checking for ```undefined```?

https://stackoverflow.com/questions/1098040/checking-if-a-key-exists-in-a-javascript-object?answertab=votes#tab-top

The current version is probably ok for this code, but it seems like a pattern we shouldn't encourage.

Reply:
Good point - 3a69de20c9

Reply:
👍